features-sdk/step_definitions/controllers-steps.js

```diff
@@ -165,3 +166,41 @@ Then('I got an error with id {string}', function (id) {
 
   assert(this.props.error.id === id, `Expected error to have id "${id}", but got "${this.props.error.id}"`);
 });
+
+When('I call the deprecated publicApi method', async function () {
+  if (process.env.KUZZLE_PROTOCOL !== 'http') {
+    return;
+  }
+
+  if (process.env.NODE_ENV !== 'development') {
```

Comment:
It would be better to add a `@development` annotation for this with the corresponding [hook](https://github.com/kuzzleio/kuzzle/blob/master/features-sdk/support/hooks.js)